src/hotspot/share/gc/g1/g1CollectedHeap.hpp

```diff
@@ -1155,15 +1155,14 @@ class G1CollectedHeap : public CollectedHeap {
     collection_set_iterate_increment_from(blk, NULL, worker_id);
   }
   void collection_set_iterate_increment_from(HeapRegionClosure *blk, HeapRegionClaimer* hr_claimer, uint worker_id);
-  // Iterate part of an array of region indexes given by offset and length, applying
+  // Iterate an array of region indexes given by offset and length, applying
```

Comment:
Since there's no `offset` in the signature, how about "Iterate over the array, `uint regions[length]`, applying ..."?

Reply:
Sure.